### PR TITLE
Introduce lint rule no_dupe_key to forbid duplicated entries in an object

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: test/functional/test-runtime.js [hunk 1]
```diff
@@ -56,7 +55,6 @@ describe('runtime', () => {
       history: {},
       navigator: {},
       setTimeout: () => {},
-      location: parseUrl('https://acme.com/document1'),
```
Comment (jridgewell) [inline test/functional/test-runtime.js]: This one would have one out.

Comment (lannka) [inline test/functional/test-runtime.js]: Seems to me the URL is not used in the test. Or did I miss something?

Comment (jridgewell) [inline test/functional/test-runtime.js]: I didn't review the tests. If it passes, LGTM.

Comment (lannka) [inline test/functional/test-runtime.js]: All tests passed. PTAL :-)

**Comment by jridgewell** [comment]
LGTM.

**Pull request merged**